test/currency-test.js

```diff
@@ -54,6 +56,16 @@ describe('Currency', function() {
       assert(r.is_valid());
       assert.strictEqual('1D2', r.to_json());
     });
+    it('from_json("1").to_human()', function() {
+      var r = currency.from_json('1');
+      assert(r.is_valid());
+      assert.strictEqual(1, r.to_json());
+    });
+    it('from_json("#$%").to_human()', function() {
+      var r = currency.from_json('#$%');
```

Comment:
Nope, turns out these are in the accepted character set https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/impl/UintTypes.cpp#L37-L43